src/containers/quiz/quiz-question/Question.constants.ts

```diff
@@ -0,0 +1,45 @@
+import { questionType } from '~/components/question-editor/QuestionEditor.constants'
+import { Question } from '~/types'
+
+interface GetQuestionStatusParams {
+  question: Question
+  answer: null | string | string[]
+  showAnswersCorrectness: boolean
+}
+
+export const isCorrectAnswer = (
+  question: Question,
+  answer: null | string | string[]
+) => {
+  if (!answer) return false
+  const { isMultipleChoice, isOpenAnswer } = questionType(question.type)
```

Comment:
`questionType` name of function is not very suitable here because it reflects value, not function. In case of naming a function, starting from verb is a common practice

Reply:
Thnaks, this function was wrote not by me but I changed it here and in other places!